packages/blitz/src/cli/utils/next-utils.ts

```diff
@@ -155,7 +155,7 @@ async function createCommandAndPort(config: ServerConfig, command: string) {
   let spawnCommand: string[] = [command]
   let availablePort: number
 
-  availablePort = await detect({port: config.port ? config.port : 3000})
+  availablePort = await detect({port: config.port})
   spawnCommand = spawnCommand.concat(["-p", `${availablePort}`])
```

Comment:
all of this should be removed :)

Reply:
@flybayer `AvailablePort` is to be removed completely? i.e, line number 156 to 159
> Sorry, I'm still learning all these😅

Reply:
yes :)

Reply:
Okay

Reply:
Please check now